ulid_uint128.hh

```diff
@@ -8,6 +8,12 @@
 #include <random>
 #include <vector>
 
+#if #if _MSC_VER > 0
```

Comment:
Typo, two `#if` statements.

Also, I don't think `__uint128_t` is available for visual studio, so maybe this file should be left unchanged?